src/components/PositionList.tsx

```diff
@@ -17,6 +17,11 @@ const PositionList: FC<PLProps> = ({ election }) => {
 
   const showDelete = user.user.id === election.manager;
 
+  //Navigate to the position application form
+  function navToPositionApplyForm(positionId:string): void{
+    history.push(`/apply/${positionId}`)
+  }
+
```

Comment:
Could you integrate this into the `onClick` on line 59? I'd say this breakout function is not neccessary for a single call.